_targets/build.project.armv7m7-imxrt106x

```diff
@@ -28,98 +30,63 @@ export PORTS_JANSSON=n
 export PORTS_CURL=n
 
 
-
 #
 # Platform dependent parameters
 #
-SIZE_PAGE=$((0x200))
-PAGE_MASK=$((0xfffffe00))
-KERNEL_OFFS=$((0x11000))
+export SIZE_PAGE=$((0x200))
```

Comment:
no need for `exporting` these variables, You're not using them in subprocess, just inside the same shell (You've sourced the file and are invoking the function). This pollutes build environment with many unnecessary variables

Applies anywhere else.
(I believe You might have done that because of shellcheck warning about unused variables but that probably might be silenced at a file-level if needed)

Reply:
Yeah, the main reason was that shellcheck warns about potential issues so that I decided to introduce `export`.